Source/WebCore/PAL/pal/text/TextCodecUserDefined.cpp

```diff
@@ -68,7 +68,7 @@ static Vector<uint8_t> encodeComplexUserDefined(StringView string, UnencodableHa
             // No way to encode this character with x-user-defined.
             UnencodableReplacementArray replacement;
```

Comment:
Maybe `UnencodableReplacementArray` needs a `span<>` operator/accessor.

Reply:
```
using UnencodableReplacementArray = std::array<char, 32>;
```

So I can actually call ` std::span { replacement }` directly. I'll fix.

Reply:
Scratch that. Even though `UnencodableReplacementArray` is a std::array, I cannot simply call `std::span { replacement }` because `replacement.size()` may differ from `replacementLength`. I don't think I can simplify here.

Reply:
Ah, that makes sense. Thank you for explaining.